contrib/endpoints/src/api_manager/service_control/testdata/allocate_quota_request_without_api_key.golden

```diff
@@ -0,0 +1,36 @@
+service_name: "test_service"
+allocate_operation {
+  operation_id: "operation_id"
+  method_name: "operation_name"
+  consumer_id: "project:project_id"
+  labels {
+    key: "servicecontrol.googleapis.com/caller_ip"
+    value: "1.2.3.4"
+  }
+  labels {
+    key: "servicecontrol.googleapis.com/referer"
+    value: "referer"
+  }
+  labels {
+    key: "servicecontrol.googleapis.com/service_agent"
+    value: "ESP/"
+  }
+  labels {
+    key: "servicecontrol.googleapis.com/user_agent"
+    value: "ESP"
+  }
+  quota_metrics {
+    metric_name: "metric_first"
+    metric_values {
+      int64_value: 1
+    }
+  }
+  quota_metrics {
+    metric_name: "metric_second"
+    metric_values {
+      int64_value: 2
+    }
+  }
+  quota_mode: BEST_EFFORT
```

Comment:
This can be a separate PR.

Just realized that quota_mode should NOT be fixed as BEST_EFFORT.
Only use BEST_EFFORT with cache and with aggregated tokens.
1) no cache case, it is NORMAL.
2) first call out of new cache item,  or negative,  it is NORMAL.